parser/src/main/kotlin/com/linkedin/dex/parser/DecodedValue.kt

```diff
@@ -67,6 +66,8 @@ sealed class DecodedValue {
                     val position = dexFile.stringIds[index].stringDataOff
                     return DecodedEnum(readStringInPosition(dexFile, position))
                 }
+                is EncodedValue.EncodedArrayValue -> return DecodedArrayValue(encodedValue.value.values.map { create(dexFile, it) }.toTypedArray())
```

Comment:
Do we need to use `return` here?
https://kotlinlang.org/docs/reference/lambdas.html#lambda-expression-syntax